todoapp_server/todos/permission.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.permissions import BasePermission
+
+class TodoAccessPermission(BasePermission):
+
+    message = 'You cannot access this todo.'
+    
+    def has_object_permission(self, request, view, obj):
+        if obj.user == request.user:
+            return True
+        else:
+            return False
```

Comment:
ovdje mozemo srtaviti ovo drugo bez else, znaci samo na kraju stavis van if-a `return False`, jer tebi ako udje u if svakaok ce na toj liniji izaci gdje imas return, a ako ne treba da vrati False

Reply:
Da inace to postujem, a ovde nisam razmisljao o tome ocigledno..